docker-qgis/utils.py

```diff
@@ -558,16 +558,14 @@ def run_workflow(
 
             feedback["steps"].append(step_feedback)
 
-        if feedback_filename in [sys.stderr, sys.stdout]:
-            print("Feedback:")
-            print(
-                json.dump(
-                    feedback,
-                    feedback_filename,
-                    indent=2,
-                    sort_keys=True,
-                    default=json_default,
-                )
+        if type(feedback_filename) == IO:
```

Comment:
Hm, even if it works, I am not sure if it is a good idea to use `typing.IO` as a operand in runtime code.

What if we use something like `isinstance(feedback_filename, io.IOBase)`?

Reply:
Good hint !